editor/src/components/canvas/canvas-strategies/relative-move-strategy.spec.browser2.tsx

```diff
@@ -0,0 +1,393 @@
+import React from 'react'
+import { createBuiltInDependenciesList } from '../../../core/es-modules/package-manager/built-in-dependencies-list'
+import { elementPath } from '../../../core/shared/element-path'
+import {
+  ElementInstanceMetadata,
+  ElementInstanceMetadataMap,
+  SpecialSizeMeasurements,
+} from '../../../core/shared/element-template'
+import {
+  canvasPoint,
+  canvasRectangle,
+  CanvasVector,
+  localRectangle,
+} from '../../../core/shared/math-utils'
+import { ElementPath } from '../../../core/shared/project-file-types'
+import { emptyModifiers } from '../../../utils/modifiers'
+import { EditorState } from '../../editor/store/editor-state'
+import { foldAndApplyCommands } from '../commands/commands'
+import {
+  getEditorState,
+  makeTestProjectCodeWithSnippet,
+  testPrintCodeFromEditorState,
+} from '../ui-jsx.test-utils'
+import { absoluteMoveStrategy } from './absolute-move-strategy'
+import { pickCanvasStateFromEditorState } from './canvas-strategies'
+import { defaultCustomStrategyState } from './canvas-strategy-types'
+import { InteractionSession, StrategyState } from './interaction-state'
+import { createMouseInteractionForTests } from './interaction-state.test-utils'
+
+const prepareEditorState = (
+  codeSnippet: string,
+  selectedViews: Array<ElementPath>,
+): EditorState => {
+  return {
+    ...getEditorState(makeTestProjectCodeWithSnippet(codeSnippet)),
+    selectedViews: selectedViews,
+  }
+}
+
+const defaultMetadata: ElementInstanceMetadataMap = {
```

Comment:
I think this was copied from a jest test (.spec.tsx) and not a karma test (.spec.browser2.tsx)
the main difference between jest tests and karma tests is that in Karma tests we don't need to fake the metadata. 

take for example https://github.com/concrete-utopia/utopia/blob/d25efb0553ad480604732d753e98087cfd87e9c9/editor/src/components/canvas/canvas-strategies/absolute-move-strategy.spec.browser2.tsx#L248-L287 where we use `await renderTestEditorWithCode` which will return with a full metadata. 

we started writing the strategies with these fake metadata tests, but we realised mid-summer (when we were refactoring some strategies) that we are much better off if we write karma tests that run the full editor, they are much more refactor-proof and should be less finicky to write too.

Reply:
the flipside (or the do not fix if it ain't broke approach) is that we can consider converting this to a proper Jest test (ie removing .browser2 from the file name), and leave it as it is. – but I suspect that we will see the shortcomings of the fake metadata once you want to test snapping for example